java/src/jmri/jmrit/beantable/LRouteTableBundle_ca.properties

```diff
@@ -1,6 +1,6 @@
-# LRouteTableBundle.properties
+# L'ItinerariTableBundle.properties
 #
-# Default properties for the jmri.jmrit.beantable.LRoute GUI elements
+# Default properties for the jmri.jmrit.beantable.L'Itinerari GUI elements
```

Comment:
Solved


El 19/08/2016 a les 18:37, Egbert Broerse ha escrit:
>
> In java/src/jmri/jmrit/beantable/LRouteTableBundle_ca.properties 
> <https://github.com/JMRI/JMRI/pull/1804#discussion_r75512951>:
>
> >  #
> > -# Default properties for the jmri.jmrit.beantable.LRoute GUI elements
> > +# Default properties for the jmri.jmrit.beantable.L'Itinerari GUI elements
>
> You shouldn't translate every place where you find the word LRoute, 
> like the file name (line 1 and this one).
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/JMRI/JMRI/pull/1804/files/c818482d84a6687ea1cfe5a77973dc0441b298c1..ac6fcec821702a36f2659b3691c95350e4ee97f1#r75512951>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AUET1U2yhdbjZoszQe0OtRFnmJjhuV4Bks5qhdu-gaJpZM4JlwXK>.
>